tests/Unleash.Tests/Internal/ErrorEvents_Tests.cs

```diff
@@ -168,7 +170,7 @@ public void CachedFilesLoader_Raises_ErrorEvent()
         }
 
         [Test]
-        public void Engine_TakeState_InvalidJson_Throws_Raises_ErrorEvent()
```

Comment:
This test is yeeted. There's no reason for this to exist, this is covered by Ygg itself. Probably my bad during the move to Ygg